### PR TITLE
[LPTOCPCI-1113] Migrate firewatch to RedHatQE

### PR DESCRIPTION
**Pull request by calebevans**
This pull request is in support of moving the firewatch repository from the CSPI-QE org to the RedHatQE org.

### CODE REVIEW

File: ci-operator/config/RedHatQE/firewatch/OWNERS
[the remark this answers is not in the document]
Comment (calebevans) [reply ci-operator/config/RedHatQE/firewatch/OWNERS:1]: This OWNERS file is specific to the firewatch folder, isn't it?

[the remark this answers is not in the document]
Comment (ascerra) [reply ci-operator/config/RedHatQE/firewatch/OWNERS:1]: yes it is! my mistake, I read this as the OWNERS file under the RedHatQE dir but that's wrong, its only the OWNERS file under firewatch dir and there is not an OWNERS file under the RedHatQE dir that we need to worry about.

Sorry about that :)

File: ci-operator/jobs/RedHatQE/firewatch/RedHatQE-firewatch-main-postsubmits.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 postsubmits:
-  CSPI-QE/firewatch:
+  RedHatQE/firewatch:
   - agent: kubernetes
     always_run: true
     branches:
     - ^main$
-    cluster: build05
+    cluster: build03
```
Comment (Simranpal) [inline ci-operator/jobs/RedHatQE/firewatch/RedHatQE-firewatch-main-postsubmits.yaml:7]: Which cluster is this

**Review by Simranpal — APPROVED** [review]
Looks good, just some of the checks were failing

File: ci-operator/jobs/RedHatQE/firewatch/RedHatQE-firewatch-main-postsubmits.yaml (hunk 1 above)
Comment (calebevans) [reply ci-operator/jobs/RedHatQE/firewatch/RedHatQE-firewatch-main-postsubmits.yaml:7]: The build cluster that is used. This is automatically decided by `make update`.

**Pull request merged**